File: app/src/main/java/DoctorDTO.kt
data class DoctorDTO(
    val id: Int,
    val name: String,
    val doctorImage: String, // URL or path to the doctor's image
    val specialization: String,
    val qualification: String,
    val experience: Int, // Number of years of experience
    val about: String,
    val email: String,
    val isAvailable: Boolean, // Availability of the doctor
    val hospital: HospitalDTO // Nested HospitalDTO object
)

File: DocVision.Api/DTO/DoctorDTO.cs
namespace FindADoctorApi.DTO
{
    public class DoctorDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string DoctorImage { get; set; }
        public string Specialization { get; set; }
        public string Qualification { get; set; }
        public int Experience { get; set; }
        public string About { get; set; }
        public string Email { get; set; }
        public bool IsAvailable { get; set; }
        public DateTime AppointmentTime { get; set; } // Time of the appointment
        public HospitalDTO Hospital { get; set; }
    }
}
